### ISSUE TITLE
Saving content with PostgreSQL

### ISSUE THREAD

**Issue by canatufkansu** [cnvs/canvas#692]
Hi,

I installed the Canvas to my existing laravel project which I first upgraded from 5.8 to 6.0 then installed auth scaffolding. Right now when go to localhost/canvas I can login and start writing a post but when I click to publish on the top left side system says "Published" but doesn't save the post to the database. Also when return to dashboard I cannot see any published posts. I'm using Postgresql and I can see all of the Canvas tables created but they are all empty. How can I find the problem ?

Thanks,

**Comment by canatufkansu** [comment]
I have just figured out that when I open developer console on Chrome, when I click save I get the following errors.

Endpoint: 2bd66cf2-b34a-4b87-b94e-d2a7137ae47a
/canvas/api/posts
```
message: "Call to a member function fill() on null"
exception: "Symfony\Component\Debug\Exception\FatalThrowableError"
file: "/Applications/MAMP/htdocs/fizbot-website-v2/vendor/cnvs/canvas/src/Http/Controllers/PostController.php"
line: 121
```

**Comment by austintoddj** [comment]
Hm interesting. [This line](https://github.com/cnvs/canvas/blob/master/src/Http/Controllers/PostController.php#L119) in the PostController is grabbing or creating a post, so I'm not sure why it would ever be null after that.

What version of the package are you currently running?

**Comment by canatufkansu** [comment]
My Laravel Framework is 6.18.3 and I just installed Canvas so it is probably the latest version. By the way I updated my comment at the top which is probably a better lead.

**Comment by austintoddj** [comment]
Much appreciated!

**Comment by thewwwizard** [comment]
New to laravel, testing blog packages. Same problem.
PostgreSQL 11.4
PHP 7.3.1
Laravel 7.4.0

**Comment by austintoddj** [comment]
Fixed in [v5.3.7](https://github.com/cnvs/canvas/releases/tag/v5.3.7)
